src/ProcessManager.php

```diff
@@ -256,7 +256,7 @@ public function shutdown($graceful = true)
         $this->output->writeln("<info>Server is shutting down.</info>");
         $this->status = self::STATE_SHUTDOWN;
 
-        $remainingSlaves = $this->slaveCount;
+        $remainingSlaves = count($this->slaves->getByStatus(Slave::READY));
```

Comment:
I'm not sure if this can happen in reality but you might also have slaves are are just booting up and require termination, too? So the remaining slaves would actually be all except for the ones already terminating?

Reply:
I have this behavior in dev env. To be honest it obscur to me ... I will retry without to see if my orchestrator respawn a new container. Could also be my local configuration making this error. But since I applied localy this patch, it's working better. 


I will take a look